packages/kit/src/views/Staking/components/StakingAmountInput.tsx

```
@@ -13,19 +14,43 @@ import { AmountInput } from '@onekeyhq/kit/src/components/AmountInput';
 import SwapPercentageStageBadge from '@onekeyhq/kit/src/views/Swap/components/SwapPercentageStageBadge';
 import platformEnv from '@onekeyhq/shared/src/platformEnv';
 
-import type { StyleProp, TextStyle } from 'react-native';
+import type {
+  NativeSyntheticEvent,
+  StyleProp,
+  TextStyle,
+  TextInputFocusEventData,
```

ESLint

[error] Member 'TextInputFocusEventData' of the import declaration should be sorted alphabetically.

(sort-imports)